src/pilot/assign_role.py

```diff
@@ -381,22 +385,31 @@ def get_raid_controller_physical_disk_ids(drac_client, raid_controller_fqdd):
         key=physical_disk_id_to_key)
 
 
-def define_storage_logical_disks(drac_client, raid_controller_name):
+def define_storage_logical_disks(drac_client, raid_controller_ids):
+    raid_controllers = []
     all_physical_disks = drac_client.list_physical_disks()
 
+    if len(raid_controller_ids) > 1:
+        for cntrl in raid_controller_ids:
+            raid_controllers.append(cntrl)
+    else:
+        raid_controllers.append(raid_controller_ids[0])
```

Comment:
It looks to me like the above code from 392 through 396 is equivalent to a just making a copy of the raid_controller_ids list.  Looking through the code below, it doesn't look like we ever modify the list, so it doesn't appear that there's any need to make a copy.  Could we just eliminate the raid_controllers variable?

Reply:
Done